src/ApiPort.VisualStudio.2015/ApiPort.VisualStudio.2015.csproj

```diff
@@ -1,8 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I noticed you removed dir.props/dir.targets from this... will the assembly still be signed?

Reply:
Directory.Build.{targets|props} is a magic file that gets included by all projects below it. See the msbuild 15 announcements for details: https://docs.microsoft.com/en-us/visualstudio/msbuild/what-s-new-in-msbuild-15-0

Reply:
Wow. that is pretty magical.